src/extensions/default/CSSCodeHints/main.js

```diff
@@ -321,7 +321,13 @@ define(function (require, exports, module) {
                 selectInitial = true;
             }
             
-            lastContext = CSSUtils.PROP_NAME;
+            if (lastContext === CSSUtils.PROP_VALUE) {
+                // close the session if we're coming from a property value
+                // see https://github.com/adobe/brackets/issues/9496
+                return false;
```

Comment:
@MarcelGerber Just a small thing. To adhere to how it is done everywhere can you change this to `return NULL`.